cli/cmd/generate.go

```diff
@@ -391,7 +391,7 @@ func addWorkloadOwnerKeyToManifest(manifst *manifest.Manifest, keyPath string) e
 func generatePolicyForFile(ctx context.Context, genpolicyPath, regoPath, policyPath, yamlPath string, logger *slog.Logger) ([32]byte, error) {
 	args := []string{
 		"--raw-out",
-		"--use-cached-files",
+		"--layers-cache-file-path=./layers-cache.json",
```

Comment:
this should be made configurable through a flag, defaulting to the file being at a default location within the configured workspaceDir.